setup.py

```diff
@@ -0,0 +1,24 @@
+import setuptools
```

Comment:
`setup.py` is (almost like it is) deprecated. I recommend switching to a `pyproject.toml` based build system.

https://packaging.python.org/en/latest/discussions/setup-py-deprecated/
https://packaging.python.org/en/latest/guides/modernize-setup-py-project/

Here's an example from me I recommend

https://github.com/Seluj78/shellhub-python/blob/main/pyproject.toml